go/arrow/compute/executor.go

```diff
@@ -44,6 +45,7 @@ type ExecCtx struct {
 	PreallocContiguous bool
 	Registry           FunctionRegistry
 	ExecChannelSize    int
+	NP                 int
```

Comment:
Maybe give it a better name, or some comments about what it is?
(I admit finding a good name is probably more than NP hard :-D)

Reply:
Renamed to `NumParallel` and added comment documentation to explain it :smile: